src/Tribe/Ticket_Object.php

```diff
@@ -211,14 +211,34 @@ class Tribe__Tickets__Ticket_Object {
 		 */
 		public $purchase_limit;
 
+		/**
+		 * Variable used to save the DateTimeZone object of the parent event
+		 *
+		 * @since TBD
+		 *
+		 * @var null
+		 */
+		private $event_timezone = null;
+
+		/**
+		 * ID of the parent event of the current Ticket
+		 *
+		 * @since TBD
+		 *
+		 * @var null
+		 */
+		private $event_id = null;
```

Comment:
If this is private, It wold be good to have a `get` method.

Reply:
This one is available already on a public method in this class as `$this->get_event()` the `private` variable was created only to store the value on memory when used in calls between the same methods of the class.